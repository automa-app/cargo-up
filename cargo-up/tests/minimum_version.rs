use insta::assert_snapshot;
mod utils;

#[test]
fn test_minimum_version() {
    let (out, _) = utils::run(
        "../fixtures/rename_member",
        &[
            "up",
            "dep",
            "upgradee",
            "--lib-path",
            "../../cargo-up",
            "--path",
            "../upgrader",
            "--name",
            "upgrader",
            "--to-version",
            "0.3.0",
        ],
    );
    assert_snapshot!(out);
}
