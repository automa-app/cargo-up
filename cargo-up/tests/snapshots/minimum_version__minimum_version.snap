---
source: tests/minimum_version.rs
expression: out
---
{
    "src/main.rs": TextEdit {
        indels: [],
    },
}

